### PR TITLE
[tmva][sofie] Fix SOFIE tests for nortcxxmodules

### PR DESCRIPTION
**Pull request by lmoneta**
This Pull request should fix a failure seen in the nortcxxmod builds

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default, ROOT-ubuntu18.04/nortcxxmod with flags -Dtmva-sofie=On

File: tmva/sofie/test/CMakeLists.txt [hunk 1]
```diff
@@ -83,8 +80,12 @@ target_compile_options(emitFromROOT PRIVATE -Wno-unused-parameter -Wno-array-bou
 
 # Automatic compilation of headers from root files
 add_custom_target(SofieCompileModels_ROOT)
-add_dependencies(SofieCompileModels_ROOT emitFromROOT)
-
+# one pcm or mudules dependency is needed for I/O converting model in ROOT file
```
Comment (sanjibansg) [inline tmva/sofie/test/CMakeLists.txt:83]: Typo error, "modules"?

**Review by sanjibansg — CHANGES_REQUESTED** [review]
LGTM!

A small typo error is there probably

File: tmva/sofie/test/CMakeLists.txt (hunk 1 above)
Comment (lmoneta) [reply tmva/sofie/test/CMakeLists.txt:83]: Thank you!

**Pull request merged**